src/lib/core/overlay/position/connected-position-strategy.ts

```diff
@@ -94,7 +94,7 @@ export class ConnectedPositionStrategy implements PositionStrategy {
    * @param element Element to which to apply the CSS styles.
    * @returns Resolves when the styles have been applied.
    */
-  apply(element: HTMLElement): Promise<void> {
+  apply(element: HTMLElement): Promise<null> {
```

Comment:
Not sure what that would look like, `null | undefined`? Otherwise I'm in favor of removing the promise completely, although I could see it being useful if we decide to introduce animations into the overlays.